app/views/mailer/mfa_notification.html.erb

```diff
@@ -0,0 +1,64 @@
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="40" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px; text-align:center">
+        We care about keeping your RubyGem.org account secure and recommend enabling multifactor authentication for <b>both UI and API</b>.
+        Setting up MFA only takes a few minutes and goes a long way towards protecting your account from any malicious actor.
+        <br/>
+        Once enabled, we will ask you for an OTP for actions like <b>sign in</b> and <b>gem push</b>.
+        <br/>
+        Please visit the link below to register a new device.
+      </div>
+
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="30" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <!-- Button -->
+      <table width="100%" border="0" cellspacing="0" cellpadding="0">
+        <tr>
+          <td align="center">
+            <table width="210" border="0" cellspacing="0" cellpadding="0">
+              <tr>
+                <td align="center" bgcolor="#e9573f">
+                  <table border="0" cellspacing="0" cellpadding="0">
+                    <tr>
+                      <td class="img" style="font-size:0pt; line-height:0pt; text-align:left" width="15"><table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="50" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+</td>
+                      <td bgcolor="#e9573f">
+                        <div class="text-btn" style="color:#ffffff; font-family:Arial, sans-serif; min-width:auto !important; font-size:16px; line-height:20px; text-align:center">
+                          <a href=<%= new_multifactor_auth_url %> target="_blank" class="link-white" style="color:#ffffff; text-decoration:none"><span class="link-white" style="color:#ffffff; text-decoration:none">ENABLE MFA</span></a>
```

Comment:
A "login/enable mfa" link in an email might smell phishy to some users. Is this a problem?

Reply:
Kind of. I don't see any alternative tho. This is better than making them search for the enable link.

Reply:
One thing might be to post an accompanying article on the RubyGems.org blog mentioning the recent incidents and why MFA is a good idea. Plus include a link to the blog post in the email.